### ISSUE TITLE
Move org.apache.cxf.xjc-utils:cxf-xjc-runtime BOM entry to quarkus-cxf-bom

### ISSUE THREAD

**Issue by ppalaga** [apache/camel-quarkus#4123]
The BOM entry
```
            <dependency>
                <groupId>org.apache.cxf.xjc-utils</groupId>
                <artifactId>cxf-xjc-runtime</artifactId>
                <version>${cxf.xjc-utils.version}</version>
                <exclusions>
                    <exclusion>
                        <groupId>jakarta.activation</groupId>
                        <artifactId>jakarta.activation-api</artifactId>
                    </exclusion>
                    <exclusion>
                        <groupId>jakarta.xml.bind</groupId>
                        <artifactId>jakarta.xml.bind-api</artifactId>
                    </exclusion>
                </exclusions>
            </dependency>
``` 

and the associated `<resolutionEntryPointInclude>org.apache.cxf.xjc-utils:cxf-xjc-runtime</resolutionEntryPointInclude>`

**Comment by ppalaga** [comment]
Blocked by https://github.com/quarkiverse/quarkus-cxf/pull/546

**Comment by zhfeng** [comment]
I think it has been fixed in `2.14.0`
